schema/user.js

```diff
@@ -5,11 +5,25 @@ const typeDef = gql`
     currency: String
     accountId: String
   }
+  
```

Comment:
These schema changes are part of WorkItem and not User

Reply:
resolved